src/components/WebinarsAndGuides/index.tsx

```diff
@@ -64,16 +69,34 @@ export const WebinarAndGuideLayout: FunctionComponent<Props> = ({
     }
     useHubSpot(hubSpotConfig)
 
+    const webinarCustomerMobileHeroStyle = { background: `url('${webinarCustomerMobileBg.src}')` }
+    const webinarCustomerHeroStyle = { background: `url('${webinarCustomerBg.src}')` }
+    const webinarProductMobileHeroStyle = { background: `url('${webinarProductMobileBg.src}')` }
+    const webinarProductHeroStyle = { background: `url('${webinarProductBg.src}')` }
+    const guideHeroStyle = { background: `url('${guideBg.src}')` }
+    const guideMobileHeroStyle = { background: `url('${guideMobileBg.src}')` }
+
+    const determineHeroStyle = (): { background: string } => {
+        if (isWebinarPg) {
+            // Customer-based Webinar
+            if (customer) {
+                return isMdOrDown ? webinarCustomerMobileHeroStyle : webinarCustomerHeroStyle
+            }
+            // Product-based Webinar
+            return isMdOrDown ? webinarProductMobileHeroStyle : webinarProductHeroStyle
+        }
+        if (!isWebinarPg) {
+            // Guides
+            return isMdOrDown ? guideMobileHeroStyle : guideHeroStyle
+        }
+        return webinarProductHeroStyle
+    }
```

Comment:
This is just using the import variables to determine which image to use:

```suggestion
    const heroImage = (): { src: string } => {
        if (isWebinarPg) {
            // Customer-based Webinar
            if (customer) {
                return isMdOrDown ? webinarCustomerMobileBg : webinarCustomerBg
            }
            // Product-based Webinar
            return isMdOrDown ? webinarProductMobileBg : webinarProductBg
        }
        if (!isWebinarPg) {
            // Guides
            return isMdOrDown ? guideMobileBg : guideBg
        }
        return webinarProductBg
    }
```